tox_core/src/state_format/old.rs

```diff
@@ -615,14 +615,16 @@ impl ToBytes for State {
 mod tests {
     use super::*;
 
-    use crate::toxcore::ip_port::*;
+    use tox_packet::ip_port::*;
 
     encode_decode_test!(
+        tox_crypto::crypto_init().unwrap(),
```

Comment:
Is it necessary?

Reply:
Sodiumoxide still needs an initialization AFAIK.

Reply:
But we shouldn't use sodiumoxide in such tests - it's just nom+cookie_factory.

Reply:
Ah, no, we generate pk/nonce.